pype/modules/ftrack/ftrack_server/lib.py

```diff
@@ -32,67 +28,10 @@
 TOPIC_STATUS_SERVER_RESULT = "pype.event.server.status.result"
 
 
-def ftrack_events_mongo_settings():
-    host = None
-    port = None
-    username = None
-    password = None
-    collection = None
-    database = None
-    auth_db = ""
-
-    if os.environ.get('FTRACK_EVENTS_MONGO_URL'):
-        result = urlparse(os.environ['FTRACK_EVENTS_MONGO_URL'])
-
-        host = result.hostname
-        try:
-            port = result.port
-        except ValueError:
-            raise RuntimeError("invalid port specified")
-        username = result.username
-        password = result.password
-        try:
-            database = result.path.lstrip("/").split("/")[0]
-            collection = result.path.lstrip("/").split("/")[1]
-        except IndexError:
-            if not database:
-                raise RuntimeError("missing database name for logging")
-        try:
-            auth_db = parse_qs(result.query)['authSource'][0]
-        except KeyError:
-            # no auth db provided, mongo will use the one we are connecting to
-            pass
-    else:
-        host = os.environ.get('FTRACK_EVENTS_MONGO_HOST')
-        port = int(os.environ.get('FTRACK_EVENTS_MONGO_PORT', "0"))
-        database = os.environ.get('FTRACK_EVENTS_MONGO_DB')
-        username = os.environ.get('FTRACK_EVENTS_MONGO_USER')
-        password = os.environ.get('FTRACK_EVENTS_MONGO_PASSWORD')
-        collection = os.environ.get('FTRACK_EVENTS_MONGO_COL')
-        auth_db = os.environ.get('FTRACK_EVENTS_MONGO_AUTH_DB', 'avalon')
-
-    return host, port, database, username, password, collection, auth_db
-
-
 def get_ftrack_event_mongo_info():
-    host, port, database, username, password, collection, auth_db = (
-        ftrack_events_mongo_settings()
-    )
-    user_pass = ""
-    if username and password:
-        user_pass = "{}:{}@".format(username, password)
-
-    socket_path = "{}:{}".format(host, port)
-
-    dab = ""
-    if database:
-        dab = "/{}".format(database)
-
-    auth = ""
-    if auth_db:
-        auth = "?authSource={}".format(auth_db)
-
-    url = "mongodb://{}{}{}{}".format(user_pass, socket_path, dab, auth)
+    url = compose_url(get_default_components())
+    database = os.environ["FTRACK_EVENTS_MONGO_DB"]
```

Comment:
I would put here
```
database = os.environ.get("FTRACK_EVENTS_MONGO_DB") or "pype"
collection = os.environ.get("FTRACK_EVENTS_MONGO_COL") or "ftrack_events"
```
To make it optional variable.

Reply:
Does that mean we should remove those variables from pype-config?

Reply:
Yes for me, but I don't want to be only who decide...

Reply:
agreed 100% variables can be optional as overrides if need be

Reply:
@mkolar is that another vote for removing those variables from pype-config?

Reply:
yes